doc/making-a-visualizer.md

```diff
@@ -101,36 +100,32 @@ sequence, so you can do sequence-dependent validation and initialization here.
     case it was left unset in the parameters dialog. Do a consistency check
     between the sequence being visualized and the parameters.
 
-#### Set up the visualizer
+#### :bulb: Set up the visualizer _(often used)_
 
 When the p5 graphics context becomes available, `setup()` is called, giving
 you your first chance to set graphics options and draw on the canvas. This is
 a good place for one-time graphics operations, like painting the background of
 a static visualizer or setting colors and stroke options that won't change
-from frame to frame. If you implement `setup()`, start by calling
-`super.setup()`, which does the behind-the-scenes work of creating a p5
-canvas.
+from frame to frame.
+
+If you implement `setup()`, start by calling `super.setup()`, which includes
+the [`createCanvas()`](https://p5js.org/reference/#/p5/createCanvas) call that
+must appear in every p5 setup function.
 
-#### _(Advanced)_ Start, stop, and dispose of the visualizer
+#### :nut*and_bolt: Show or stop the visualization; depart from a page element *(advanced)\_
```

Comment:
same formatting issues

Reply:
OK, the stray asterisk is gone; resolving.